configure.ac

```diff
@@ -456,8 +456,12 @@ case "$enable_hwloc" in
    no)
       ;;
    yes)
-      AC_CHECK_LIB([hwloc], [hwloc_get_proc_cpubind], [], [AC_MSG_ERROR([can not find required library libhwloc])])
-      AC_CHECK_HEADERS([hwloc.h], [], [AC_MSG_ERROR([can not find require header file hwloc.h])])
+      PKG_CHECK_MODULES(HWLOC, hwloc,
```

Comment:
Please check for `PKG_CHECK_MODULES` to be available, like is done for e.g. `enable_delayacct`, as there are platforms where `pkgconfig` is not available.

Reply:
Thanks for your feedback, I updated the PR.